### DISCUSSION TITLE
Is there an alternative that supports `Vue2` with `Vite`?

### DISCUSSION THREAD

**Discussion by He110te4m** [vuepress/core#1063] [Q&A] [answered]
I have a relatively large `Vue2` project that has been upgraded to `Vite`, but neither `VuePress2` nor `VitePress` support `Vue2`, is there an alternative to support `Vue2` with `Vite`? After seeing the speed of Vite, I don't want to put up with VuePress1's webpack anymore.

**Comment by Mister-Hope** [comment, marked as answer]
I think there is not.

You may fork vuepress v1 and implement it by your own. Develpers especially open source contributors probably do not want to waste time on outdated packages.
